### DISCUSSION TITLE
Stream port unix socket / IP address input

### DISCUSSION THREAD

**Discussion by dgcampea** [jcorporation/myMPD#945] [Q&A]
myMPD version: 10.1.6

**Describe the bug**

MPD httpd output can bind to UNIX domain sockets or specific IP addresses but myMPD only allows choosing a port.

**Expected behavior**

Allow passing either a path to a socket file or address + ip.

**Server plattform (please complete the following information):**
 - OS: GuixSD
 - OS version: 7c476873e0300711bf92668cf01abd28f7295ead
 - Plattform: x86_64
 - MPD version: 0.23.11

**Sample MPD configuration block:**
```
audio_output {
name "HTTP - Opus"
type "httpd"
enabled "yes"
tags "yes"
always_on "yes"
mixer_type "none"
encoder "opus"
bitrate "192000"
signal "music"
format "48000:16:2"
bind_to_address "/var/run/mpd/http-opus.socket"
max_clients "0"
}
```

**Comment by jcorporation** [comment]
myMPD calculates the stream port from the mpd host setting. If you use none-standard configuration, you can use the stream uri setting.

Mongoose, the embedded web server and http client, does not support unix domain sockets.
